js/build.js

```diff
@@ -1,185 +1,99 @@
-      */
 
-      Fliplet.Widget.initializeChildren(this, vm);
-      resolve(vm);
+      container.id = data.id;
+      dynamicContainerInstances[data.id] = container;
+    }, {
+      supportsDynamicContext: true
     });
-
-    dynamicContainerInstances.push(container);
-  }, {
-    supportsDynamicContext: true
   });
-});
 
-Fliplet.DynamicContainer.get = function(name, options) {
-  options = options || { ts: 10 };
+  Fliplet.DynamicContainer.get = function(name, options) {
+    options = options || { ts: 10 };
 
-  return Fliplet().then(function() {
-    return Promise.all(dynamicContainerInstances).then(function(containers) {
-      var container;
+    return Fliplet().then(function() {
+      return Promise.all(_.values(dynamicContainerInstances)).then(function(containers) {
+        var container;
 
-      if (typeof name === 'undefined') {
-        container = containers.length ? containers[0] : undefined;
-      } else {
-        containers.some(function(vm) {
-          if (vm.name === name) {
-            container = vm;
+        if (typeof name === 'undefined') {
+          container = containers.length ? containers[0] : undefined;
+        } else {
+          containers.some(function(vm) {
+            if (vm.name === name) {
+              container = vm;
 
-            return true;
-          }
-        });
-      }
-
-      if (!container) {
-        if (options.ts > 5000) {
-          return Promise.reject('Container not found after ' + Math.ceil(options.ts / 1000) + ' seconds.');
+              return true;
+            }
+          });
         }
 
-        // Containers can render over time, so we need to retry later in the process
-        return new Promise(function(resolve) {
-          setTimeout(function() {
-            options.ts = options.ts * 1.5;
+        if (!container) {
+          if (options.ts > 5000) {
+            return Promise.reject('Container not found after ' + Math.ceil(options.ts / 1000) + ' seconds.');
+          }
 
-            Fliplet.DynamicContainer.get(name, options).then(resolve);
-          }, options.ts);
-        });
-      }
+          // Containers can render over time, so we need to retry later in the process
+          return new Promise(function(resolve) {
+            setTimeout(function() {
+              options.ts = options.ts * 1.5;
 
-      return container;
+              Fliplet.DynamicContainer.get(name, options).then(resolve);
+            }, options.ts);
+          });
+        }
+
+        return container;
+      });
     });
-  });
-};
+  };
 
-Fliplet.DynamicContainer.getAll = function(name) {
-  return Fliplet().then(function() {
-    return Promise.all(dynamicContainerInstances).then(function(containers) {
-      if (typeof name === 'undefined') {
-        return containers;
-      }
+  Fliplet.DynamicContainer.getAll = function(name) {
+    return Fliplet().then(function() {
+      return Promise.all(_.values(dynamicContainerInstances)).then(function(containers) {
```

Comment:
[MEDIUM] @tonytlwu, since there's the `Object.values` native method, there's no point in utilizing the lodash `_.values` method, especially when the native method is over 21% more perfomant.

https://www.measurethat.net/Benchmarks/Show/9112/0/lodash-values-vs-objectvalues
<img width="747" alt="image" src="https://github.com/Fliplet/fliplet-widget-dynamic-container/assets/144125903/68e658f7-bc33-432a-87e2-352494a25514">

Reply:
Updates committed.